src/main/java/com/ucm/ms/accounts/controllers/UserAccountsController.java

```diff
@@ -53,17 +56,15 @@ public ModelAndView confirmUserAccount(ModelAndView modelAndView, @RequestParam(
         return modelAndView;
     }
     
-    /**
```

Comment:
So the thing with this one is you're deleting a method that functionally isn't replaced by the other one. Keeping this controller method in here returns all user accounts for all users. We should keep this one.